src/imfilter.jl

```diff
@@ -1547,7 +1547,7 @@ function normalize_separable!(r::AbstractResource, A, kernels::NTuple{N,TriggsSd
     function imfilter_inplace!(r, a, kern, border)
         imfilter!(r, a, a, (kern,), border)
     end
-    filtdims = ntuple(d->imfilter_inplace!(r, fill(1, inds[d]), kernels[d], border), Val(N))
+    filtdims = ntuple(d->imfilter_inplace!(r, fill(1.0, inds[d]), kernels[d], border), Val(N))
```

Comment:
Since this is inplace I think switching to 1.0 would be OK. Is this overflow prevention or something else?

Reply:
IIRC there was an "inexact" error for one of the tests without this.